src/frontends/tensorflow/src/op/approximate_equal_op.cpp

```diff
@@ -0,0 +1,37 @@
+#include "common_op_table.hpp"
+#include "openvino/op/abs.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/select.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils.hpp"
+#include "openvino/frontend/tensorflow/node_context.hpp"
+#include "openvino/frontend/tensorflow/variable.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_approximate_equal_op(const NodeContext& node) {
+        default_op_checks(node, 2, {"ApproximateEqual"});
+        auto x = node.get_input(0);
+        auto y = node.get_input(1);
+        auto tolerance = make_shared<v0::Constant>(element::i32, Shape{}, node.get_input(2));
```

Comment:
tolerance is an attribute, not input. Please retrieve its value using `node.get_attribute`, see examples of translators for other operations.

Reply:
@rkazants surely will update it. Thank you

Reply:
@rkazants please review addressed the changes.